core/frontend/src/tile/OrbitGtTileTree.ts

```diff
@@ -359,6 +359,10 @@ export namespace OrbitGtTileTree {
 
   async function initializeBlobStorage(_props: OrbitGtBlobProps, iModel: IModelConnection, _modelId: Id64String) {
 
+    // rdsUrl must be defined and not null to be resolved
+    if(_props.rdsUrl == null)

```

Comment:
The linter's going to complain about `==` vs `===`, and `===` won't catch `undefined`.
If you would consider an empty string also unresolvable, consider replacing with `if (!_props.rdsUrl)`.

Reply:
I've changed it to !rdsUrl, although it wasn't causing an issue in the linter locally, I suspected it was configured to permit == with null.